source/precalculus/source/06-TR/01.ptx

```diff
@@ -12,6 +12,308 @@
+
+        </p>
+      </introduction>
+      <task>
+        <statement>
+        <p> <figure xml:id="fig-90-degree-angle"><image width="50%">
+          <sageplot>
+            p=arrow((0,0),(3,3),aspect_ratio=1)
+            p+=arrow((0,0),(-3,3))
+            p+=arc((0,0),0.5,sector=(pi/4, 3*pi/4),color="black")
+            p.axes(False)
+            p
+          </sageplot>
+        </image></figure>
+          <ol marker="A." cols="2">
+            <li><p><m>45^{\circ}</m> </p></li>
+            <li><p> <m>90^{\circ}</m> </p></li>
+            <li><p> <m>135^{\circ}</m> </p></li>
+            <li><p> <m>180^{\circ}</m></p></li>
+        </ol> </p>
+        </statement>
+        <answer>
+          <p>
+            B
+          </p>
+        </answer>
+      </task>
+      <task>
+        <statement>
+        <figure><image width="50%">
+          <sageplot>
+            p=arrow((0,0),(-4,1),aspect_ratio=1)
+            p+=arrow((0,0),(4,-1))
+            p+=point((0,0),size=50)
+            p+=arc((0,0),0.5,sector=(atan(-1/4), atan(-1/4)+pi),color="black")
+            p.axes(False)
+            p
+          </sageplot>
+        </image></figure>
+        <p>
+          <ol marker="A." cols="2">
+            <li><p><m>45^{\circ}</m> </p></li>
+            <li><p> <m>90^{\circ}</m> </p></li>
+            <li><p> <m>135^{\circ}</m> </p></li>
+            <li><p> <m>180^{\circ}</m></p></li>
+        </ol> </p>
+        </statement>
+        <answer>
+          <p>
+            D
+          </p>
+        </answer>
+      </task>
+      <task>
+        <statement>
+        <p> <figure xml:id="fig-135-degree-angle"><image width="50%">
+          <sageplot>
+            p=arrow((0,0),(0,4),aspect_ratio=1)
+            p+=arrow((0,0),(-3,-3))
+            p+=arc((0,0),0.5,sector=(pi/2, 5*pi/4),color="black")
+            p.axes(False)
+            p
+          </sageplot>
+        </image></figure>
+          <ol marker="A." cols="2">
+            <li><p><m>45^{\circ}</m> </p></li>
+            <li><p> <m>90^{\circ}</m> </p></li>
+            <li><p> <m>135^{\circ}</m> </p></li>
+            <li><p> <m>180^{\circ}</m></p></li>
+        </ol> </p>
+        </statement>
+        <answer>
+          <p>
+            C
+          </p>
+        </answer>
+      </task>
+    </activity>
+    <definition>
+      <statement>
+        <p>
+          An angle is in <term>standard position</term> if its vertex is located at the origin and its initial side extends along the positive <m>x</m>-axis. 
+        </p>
+      </statement>
+      <figure xml:id="standard-position"><image width="50%">
+        <sageplot>
+          p=arrow((0,0),(4,0),aspect_ratio=1)
+          p+=arrow((0,0),(-3,-3))
+          p+=arc((0,0),0.5,sector=(0, 5*pi/4),color="black")
+          p
+        </sageplot>
+      </image></figure>
```

Comment:
@StevenClontz We want to have the axes show up on this one, but no tickmarks or tickmark labels.  Do you know how to do that?

Reply:
I think I figured it out. The key is to past a list of two empty lists (`ticks=[[],[]]`), rather than `ticks=None` like I was trying.

```suggestion
      <figure xml:id="standard-position"><image width="50%">
        <sageplot>
          p=arrow((0,0),(4,0),aspect_ratio=1,ticks=[[],[]])
          p+=arrow((0,0),(-3,-3))
          p+=arc((0,0),0.5,sector=(0, 5*pi/4),color="black")
          p
        </sageplot>
      </image></figure>
```